README.md

```diff
@@ -97,138 +97,147 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
-### 0.73 - August 2023 Update
+### 0.74 - September 2023 Update
 
-In this release, we focused on releasing new features, stability and improvements.
+In this release, we focused on stability and improvements.
 
 **Highlights**
 
- - Keyboard manager now allows Numpad keys to be bound separately from the main keyboard. If using a previous configuration, only the main keyboard (not numpad) keys will be mapped. To restore the previous behavior, manually add separate bindings for numpad keys.
- - New utility: Crop And Lock allows you to crop a current application into a smaller window or just create a thumbnail. Focus the target window and press the shortcut to start cropping.
- - FancyZones code improvements and refactor.
- - Modernized ImageResizer UX.
- - PowerRename advanced counter functionality.
+ - Upgraded to Windows App SDK 1.4.1, increasing stability of WinUI3 utilities. Thanks [@dongle-the-gadget](https://github.com/dongle-the-gadget) for starting the upgrade!
+ - Text Extractor was upgraded to its version 2.0, with a new overlay, table mode and more Quality of Life improvements. Thanks [@TheJoeFin](https://github.com/TheJoeFin)!
+ - Improved FancyZones stability, fixing some layout resets and improving handling of newly created windows on Windows 11.
+ - Fixed many silent crashes that were reported to Watson and the user's event viewer.
 
 ### General
 
- - Added missing CoUninitialize call in elevation logic. Thanks [@sredna](https://github.com/sredna)!
- - New utility: Crop And Lock. Thanks [@robmikh](https://github.com/robmikh)! and [@kevinguo305](https://github.com/kevinguo305)!
- - Added new /helped fabric bot command to GitHub repo. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
- - Fixed crashes caused by invalid settings. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+ - Turning animations off in Windows Settings will now also turn them off in PowerToys.
+ - Upgraded the Windows App SDK dependency to 1.4.1. Thanks [@dongle-the-gadget](https://github.com/dongle-the-gadget) for the original 1.4.0 upgrade!
+ - Show in the thumbnail label and application titles when running as administrator. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+ - Upgraded the Win UI Community Toolkit dependency to 8.0. Thanks [@niels9001](https://github.com/niels9001)!
 
-### Always On Top
+### Awake
 
- - Added border transparency.
+ - Added down-sampled variants to the application's icon. Thanks [@morriscurtis](https://github.com/morriscurtis)!
 
-### FancyZones
+### Color Picker
 
- - Fixed issue causing canvas zones being drawn only when dragging in the zone area.
- - Fixed user-defined default layout highlighting issue.
- - Refactored and improved code quality.
- - Fixed issue causing the wrong layout to be applied when duplicating a non-selected layout.
+ - After selecting a new color in the editor, the history will scroll to the top. Thanks [@peerpalo](https://github.com/peerpalo)!
```

Comment:
```suggestion
 - After adding a new color in the editor, the history will the new color into view. Thanks [@peerpalo](https://github.com/peerpalo)!
```

Reply:
@Jay-o-Way , thanks for the suggestion, but "the history will the new color into view" doesn't make much sense.

Reply:
missed a word - updated